server/controllers/user.js

```diff
@@ -0,0 +1,83 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const userModel = require('../model/user');
+
+const app = express();
+
+const Logger = require('../services/logger_services');
+
+const logger = new Logger('app');
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+const getUser = (req, res) => {
+  const id = Number(req.params.userId);
+  const user = userModel.getUser(id);
+  logger.setLogData(user);
+  logger.info('You passed a user ID');
+  return res.status(200).json({
+    message: 'You passed a user ID',
+    user,
+  });
+};
```

Comment:
In general, (not specifically in this function but also here) you should handle all kind of errors and return it to the client.
for instance, in this function what happens if the userId provided not found?

fix in all relevant places.